doc/barman.md

```diff
@@ -0,0 +1,136 @@
+## Point in time recovery with Barman
+
+[Barman](http://www.pgbarman.org) (Backup and Recovery Manager) is an open-source administration tool for disaster recovery of PostgreSQL servers.
+
+This example shows how to integrate Barman [Barman](http://www.pgbarman.org) with Stolon. Therefore, by integrating Barman with Stolon,
+you can dramatically reduce RTO(Recovery Time Objective) and RPO(Recovery Point Objective) to nearly zero values.
+
+### Introduction
+
+Offers two options for backup and archiving. Traditionally, Barman has supported standard WAL file shipping through
+PostgreSQL’s `archive_command` (usually via `rsync/SSH`). With this method, WAL files are archived only when PostgreSQL switches to a new WAL file.
+To keep it simple, this normally happens every 16 MB worth of data changes. Barman also introduces streaming of WAL files for PostgreSQL 9.2 or later,
+as an additional method for transactional log archiving through `pg_receivexlog`. WAL streaming is able to reduce the risk of data loss,
+bringing RPO down to nearly zero values. In addition, Barman introduces support for replication slots with PostgreSQL servers 9.4 or later, therefore allowing WAL streaming-only configurations.
+Moreover, you can now add Barman as a synchronous WAL receiver in your PostgreSQL 9.5 (or later) cluster and achieve zero data loss (RPO=0).
+Thus, the traditional archiving of WAL logs can lead to data loss in the event of an accident.
+
+### Possible solution for integration of Stolon with Barman
+
+![Stolon architecture with Barman](stolon_and_barman.png)
```

Comment:
I'll remove this image since it's not updated and will be difficult to keep it updated in the future:

* etcd references
* looks like rsync is needed
* direct connections to an instance instead of using the stolon-proxy